parser-typechecker/src/Unison/Server/Types.hs

```diff
@@ -54,6 +54,8 @@ type APIGet c = Get '[JSON] (APIHeaders c)
 
 type HashQualifiedName = Text
 
+type NamespaceFQN = Text
```

Comment:
Used this over `HashQualifiedName` for namespace params. In the future, I bet we could get these parsed by Servant.